contracts/workflows/RoyaltyWorkflows.sol

```diff
@@ -131,7 +129,7 @@ contract RoyaltyWorkflows is IRoyaltyWorkflows, MulticallUpgradeable, AccessMana
             returns (uint256 claimedAmount) {
                 amountsClaimed[i] += claimedAmount;
             } catch {
-                amountsClaimed[i] += 0;
+                // Continue to the next currency token
```

Comment:
Should we revert if there are some error happen?

Reply:
In my opinion, we shouldn't revert this change.

Consider a scenario where an IP owner has unclaimed snapshots `snapshotA` and `snapshotB`, which only contain revenue tokens `tokenA` and `tokenB`. There's also a `tokenC` that hasn't been included in any snapshots before. If the user wants to claim `tokenC` along with their existing snapshots, they would provide `tokenC` in `royaltyClaimDetails`.

If we choose to revert in this case, the user's claim request will fail. However, I think this is a valid request and should succeed.

Reply:
Agreed, it is valid use case we should support. 

In that case, it would better only catch the specific error but all errors, so that it failed explicitly if unexpected error happens.  
In best practice, it would better also return which tokens have been claimed successfully.

Reply:
Resolved in [45ec772](https://github.com/storyprotocol/protocol-periphery-v1/pull/75/commits/45ec7727237791ad0e01e8f2d2e83d735fe3be80) :)